src/service/standard-actions-impl.js

```diff
@@ -88,14 +103,62 @@ export class StandardActions {
    */
   handleHide(invocation) {
     const target = dev().assertElement(invocation.target);
+
     this.resources_.mutateElement(target, () => {
       if (target.classList.contains('i-amphtml-element')) {
         target./*OK*/collapse();
       } else {
         toggle(target, false);
+        target.setAttribute('hidden', '');
       }
     });
   }
+
+  /**
+   * Handles "show" action. This is a very simple action where "display: none"
+   * is removed from the target element.
+   * @param {!./action-impl.ActionInvocation} invocation
+   */
+  handleShow(invocation) {
+    const target = dev().assertElement(invocation.target);
+
+    if (getMode().development) {
```

Comment:
Sorry, actually removing `getMode().development` check and using the `user().warn` is the right approach.  `getMode().development` won't help external developers. make sure you `return` after `user().warn`, unlike `assert` it does not throw synchronously to stop execution.